apps/site/assets/css/_events.scss

```diff
@@ -40,3 +42,30 @@
 .event-inline-block {
   display: inline-block;
 }
+
+.m-event__date-circle {
+  display: flex;
+  flex-direction: column;
+  height: 4rem;
+  width: 4rem;
+  background-color: $gray-bordered-background;
+  border-radius: 2rem;
+  justify-content: center;
+  align-items: center;
+  margin-right: 1.5rem;
+  text-transform: uppercase;
+  color: $gray;
+  @include media-breakpoint-down(xs) {
+    display: none;
+  }
+}
+.m-date-range {
+  font-size: .88rem;
```

Comment:
This font-size is suuuuuper close to `$font-size-small-caps` - would it make sense to use that one here?